### PR TITLE
refactor(components): Rename StoreMessage to MessageStore for Better Clarity

### PR DESCRIPTION
**Pull request by Vigtu**
# Changes in Message Store Component:

## Component Rename:
1. StoreMessage -> MessageStore:
- Renamed class from StoreMessageComponent to MessageStoreComponent
- Updated display name to "Message Store"
- Changed icon from "save" to "message-square-text"
- Updated component name to "MessageStore"

## Output Configuration:
- Added hidden=True to stored_messages output

## Import Updates:
- Updated import in __init__.py:
  * store_message -> message_store
  * StoreMessageComponent -> MessageStoreComponent

## File Rename:
- store_message.py -> message_store.py

## No Functional Changes:
- Core functionality remains the same
- Only naming and cosmetic changes applied
- Component description maintained

### CODE REVIEW

**Comment by erichare** [comment]
@Vigtu same for this one, if you don't mind fixing the merge conflict i'll do a review!

File: src/backend/base/langflow/components/helpers/message_store.py [hunk 1]
```diff
@@ -7,11 +7,11 @@
...
 
-class StoreMessageComponent(Component):
-    display_name = "Store Message"
+class MessageStoreComponent(Component):
+    display_name = "Message Store"
     description = "Stores a chat message or text into Langflow tables or an external memory."
-    icon = "save"
-    name = "StoreMessage"
+    icon = "message-square-text"
+    name = "MessageStore"
```
Comment (ogabrielluiz) [inline src/backend/base/langflow/components/helpers/message_store.py:14]: This breaks backwards compatibility

**Review by ogabrielluiz — CHANGES_REQUESTED** [review]
Hey @Vigtu 

Thanks for the PR! 

When changing componentes we shouldn't change the file name AND the name. You must keep one as it is for backwards compatibility. 

You should be better off reverting the change to the name attribute. 

Also, if you could add tests to this component like you can see in src/backend/tests/unit/components/../test_input_components.py it would be really good. 


Thanks!

**Review by ogabrielluiz — APPROVED** [review]
LGTM

**Review by erichare — APPROVED** [review]

**Pull request merged**